src/pages/core/guides/javascript/how-to-create-a-core-nft-asset.md

```diff
@@ -0,0 +1,371 @@
+---
+title: How to Create a Core NFT Asset
+metaTitle: How to Create a Core NFT Asset
+description: Learn how to create a Core NFT Asset on the Solana blockchain with the Metaplex Core package.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-18-2024'
+---
+
+This is an intial guide on how to create a next-gen Digital Asset Nft (Core Asset) on the Solana blockchain using the Metaplex Mpl Core protocol. Mpl Core Assets provide the next wave of NFT projects on Solana with greater creativity and a broad dynamic experiance for both creators and owners with Mpl Core's simplied design and unique plugin system.
+
+## Prerequisite
+
+- Code Editor of your choice (recommended Visual Studio Code)
+- Node 18.x.x or above.
+
+## Initial Setup
+
+This guide will run through create of an Nft Core Asset with Javascript based on a single file script. You may need to modify and move functions around to suit your needs.
```

Comment:
There's still several places on this particular page where it is called "Nft" instead of "NFT" and should be updated.  Looks like fixed everywhere else though.